package.json

```
@@ -7,9 +7,9 @@
     "start": "sirv public"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^16.0.0",
-    "@rollup/plugin-node-resolve": "^10.0.0",
-    "rollup": "^2.3.4",
+    "@rollup/plugin-commonjs": "^17.0.0",
+    "@rollup/plugin-node-resolve": "^11.0.0",
+    "rollup": "^2.0.0",
```

I don't want to change this in case `2.3.4` was there to indicate that's a minimum version that works

```suggestion
    "rollup": "^2.3.4",
```